src/managed_data/_api.py

```diff
@@ -103,7 +103,7 @@ def make_database_rows(
         database_id: ID or Human ID of the database
 
     Returns:
-        A dictionary that conforms to a [EnsureRowsResponse][src.managed_data.schema.EnsureRowsResponse]
+        A dictionary that conforms to a EnsureRowsResponse
```

Comment:
this model doesn't exist yet. i don't want to make any more models because i anticipate all this code will get thrown away when we auto-generate the low level code and models